services/gemma-local/streaming.py

```
@@ -4,106 +4,93 @@
 import time
 import uuid
 import logging
-from typing import AsyncGenerator, Optional
-import torch
+from typing import AsyncGenerator, Optional, Union, List
```

Ruff (0.13.1)

Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)